### PR TITLE
docs:Updated integration docs structure for chat/google_vertex_ai_palm

### PR DESCRIPTION
**Pull request by manokhina**
Description: 

- checked that the doc chat/google_vertex_ai_palm is using new functions: invoke, stream etc. 
- added Gemini example
- fixed wrong output in Sanskrit example

Issue: https://github.com/langchain-ai/langchain/issues/15664
Dependencies: None
Twitter handle: None

### CODE REVIEW

File: docs/docs/integrations/chat/google_vertex_ai_palm.ipynb [hunk 1]
```diff
@@ -148,13 +152,14 @@
    },
    "source": [
     "## Code generation chat models\n",
-    "You can now leverage the Codey API for code chat within Vertex AI. The model name is:\n",
-    "- codechat-bison: for code assistance"
+    "You can now leverage the Codey API for code chat within Vertex AI. The models available are:\n",
+    "- codechat-bison: for code assistance\n",
+    "- gemini-pro: Google's most capable and general model"
```
Comment (lkuligin) [inline docs/docs/integrations/chat/google_vertex_ai_palm.ipynb:157]: I'm not sure we should mention gemini-pro here.

File: docs/docs/integrations/chat/google_vertex_ai_palm.ipynb [hunk 2]
```diff
@@ -57,8 +56,10 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
```
Comment (lkuligin) [inline docs/docs/integrations/chat/google_vertex_ai_palm.ipynb:60]: can we add a specific note that `ChatVertexAI` support Gemini and Palm on top of the documentation page, please?

File: docs/docs/integrations/chat/google_vertex_ai_palm.ipynb [hunk 3]
```diff
@@ -205,12 +200,14 @@
    "source": [
     "## Asynchronous calls\n",
     "\n",
-    "We can make asynchronous calls via the Runnables [Async Interface](/docs/expression_language/interface)"
+    "We can make asynchronous calls via the Runnables [Async Interface](/docs/expression_language/interface).\n",
+    "\n",
+    "Note: System message is not yet supported by Gemini models."
```
Comment (lkuligin) [inline docs/docs/integrations/chat/google_vertex_ai_palm.ipynb:205]: we released a [partial] support for a system message yesterday, could you update this piece, please?

File: docs/docs/integrations/chat/google_vertex_ai_palm.ipynb [hunk 4]
```diff
@@ -18,6 +18,13 @@
     "\n",
     "Note: This is separate from the Google PaLM integration. Google has chosen to offer an enterprise version of PaLM through GCP, and this supports the models made available through there. \n",
     "\n",
+    "ChatVertexAI exposes all foundational models available in Google Cloud:\n",
+    "\n",
+    "- Gemini (`gemini-pro` and `gemini-pro-vision`)\n",
+    "- PaLM 2 for Text (`text-bison`)\n",
+    "- Codey for Code Generation (`codechat-bison`)\n",
+    "For a full and updated list of available models visit VertexAI documentation\n",
```
Comment (lkuligin) [inline docs/docs/integrations/chat/google_vertex_ai_palm.ipynb:26]: nits: should we add a link here?

File: docs/docs/integrations/chat/google_vertex_ai_palm.ipynb (hunk 3 above)
Comment (lkuligin) [reply docs/docs/integrations/chat/google_vertex_ai_palm.ipynb:388]: I'm not sure it was updated, am I missing anything?

Comment (manokhina) [reply docs/docs/integrations/chat/google_vertex_ai_palm.ipynb:388]: it was added by someone else, already in master, that's why it's not shown here in diff
https://github.com/manokhina/langchain/blob/master/docs/docs/integrations/chat/google_vertex_ai_palm.ipynb

**Review by lkuligin — APPROVED** [review]

**Review by efriis — APPROVED** [review]
Thanks!

**Review by baskaryan — APPROVED** [review]

**Pull request merged**